docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-powershell-minidump-script.asciidoc

```diff
@@ -0,0 +1,141 @@
+[[prebuilt-rule-0-16-2-powershell-minidump-script]]
+=== PowerShell MiniDump Script
+
+This rule detects PowerShell scripts capable of dumping process memory using WindowsErrorReporting or Dbghelp.dll MiniDumpWriteDump. Attackers can use this tooling to dump LSASS and get access to credentials.
```

Comment:
```suggestion
This rule detects PowerShell scripts capable of dumping process memory using WindowsErrorReporting or `Dbghelp.dll` MiniDumpWriteDump. Attackers can use this tooling to dump Local Security Authority Server Service (LSASS) memory and get access to credentials.
```

Reply:
Flagging this because I added "memory" to the second sentence--looked as though it might've been missing, so ignore if it wasn't.